src/protocol/protocol.c

```diff
@@ -121,6 +121,7 @@ void PROTOCOL_DeInit()
     CLOCK_StopTimer();
     if(Model.protocol != PROTOCOL_NONE && PROTOCOL_LOADED)
         PROTO_Cmds(PROTOCMD_DEINIT);
+    CLOCK_StartMixer(); // run mixer on timer so channels are updated for things like calibration
```

Comment:
I prefer putting the comments above the line, which is more common practice and also make the width of line small.

Reply:
I actually prefer this way because it makes it more likely the comment will be preserved if the line is moved.  But it isn't a strong rule